services/core-web/src/components/mine/Permit/Search/components/MarkdownViewer.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import ReactMarkdown from 'react-markdown';
+
+interface MarkdownViewerProps {
+    markdown: string;
+}
+
+const MarkdownViewer: React.FC<MarkdownViewerProps> = ({ markdown }) => {
+    return (
+        <div className="permit-search__markdown">
```

Comment:
Tiny thing, but since we're making this component nicely generic, it might be good make the className a bit more generic as well.